Apps/FCM/android/app/src/main/AndroidManifest.xml

```diff
@@ -56,6 +56,7 @@
             android:resource="@drawable/ic_notification" />
         <meta-data
             android:name="com.google.firebase.messaging.default_notification_color"
-            android:resource="@color/notification_icon" />
+            android:resource="@color/notification_icon"
+            tools:replace="android:resource" />
```

Comment:
Don't we expect a similar change in APN app's  `AndroidManifest` file ?

Reply:
No, because the problem arises from `react-native-firebase` which is only added to FCM app. Also, APN Android app has other issues at the moment too. You can find more details in [slack thread](https://customerio.slack.com/archives/C06H7MMLH6C/p1708523823792659) I posted earlier.